### PR TITLE
getting started page

### PR DESCRIPTION
**Pull request by LucasHT22**
Creates a getting started page, an introduction to Sprig

### CODE REVIEW

File: src/pages/sprikipedia/getting-started/acknowledgements.astro [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// spriki19
```
Comment (grymmy) [inline src/pages/sprikipedia/getting-started/acknowledgements.astro:1]: what is this?

Comment (LucasHT22) [reply src/pages/sprikipedia/getting-started/acknowledgements.astro]: per our discussion, removed

**Comment by grymmy** [comment]
> _ No description provided. _

please add a good description to this PR that describes what it does

**Review by grymmy — CHANGES_REQUESTED** [review]
see comment

**Comment by LucasHT22** [comment]
Done

**Review by grymmy — APPROVED** [review]

**Pull request merged**